### ISSUE TITLE
make install fails: file INSTALL cannot find version.h

### ISSUE THREAD

**Issue by ryandesign** [fletcher/MultiMarkdown-6#147]
`make install` fails with multimarkdown 6.4.0 on macOS:

```
CMake Error at cmake_install.cmake:82 (file):
  file INSTALL cannot find
  "/path/to/Sources/libMultiMarkdown/include/version.h".
make: *** [install/fast] Error 1
```

It looks like version.h is getting created in the root of the build directory, not in /path/to/Sources/libMultiMarkdown/include.

The workaround we are using in MacPorts is to copy version.h from the build directory back to /path/to/Sources/libMultiMarkdown/include prior to running `make install`, but ideally that would not be necessary.

**Comment by fletcher** [comment]
How are you building it?  It works fine for me.

**Comment by ryandesign** [comment]
FYI this problem remains in version 6.5.0.

**Comment by fletcher** [comment]
And I still can't replicate it.  Not sure what to tell you, but you're doing something wrong.

I did just notice that you mentioned using `-DSHAREDBUILD=1` -- is there a reason for this?

**Comment by rsumner868** [comment]
```
```

```
```

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rsumner868** [comment]
```

```

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fletcher** [comment]
@rsumner868  -- your comments are not coming through correctly.

**Comment by ryandesign** [comment]
> I did just notice that you mentioned using `-DSHAREDBUILD=1` -- is there a reason for this?

Thanks for pointing that out again.

When I updated multimarkdown to 6.5.0 in MacPorts, I reexamined all of the bug reports I had submitted to see which ones still applied. Per your advice in #145, I stopped using `-DSHAREDBUILD=1`.

I thought I had verified that this issue still occurred after doing that, however it is now clear that I did not. I have now confirmed that like #145 this is another issue that only occurs when using `-DSHAREDBUILD=1`.

**Comment by fletcher** [comment]
The `SHAREDBUILD` option is only there for people who specifically need it in order to build a shared library for other projects.  But given that MultiMarkdown is pretty small, it's probably not worth doing this except in specific circumstances.

Glad removing that option fixes things.

**Comment by ryandesign** [comment]
I understand. For those users who do want the shared library, you may want to fix this build problem and #145.
